src/utils.ts

```diff
@@ -272,49 +289,59 @@ export function addDefaultArgs(Cypress, args, opts = {}) {
   return newArgs;
 }
 
-process.env.FORCE_COLOR = true;
+(process as any).env.FORCE_COLOR = true;
```

Comment:
This is add that you had to cast this as any. I think its because you need to install `@types/node` and include in the types `"node"`

Reply:
Thanks for pointing that out, I'll give it a try

Reply:
I think it is because I was setting an environment variable to a boolean when the type should be either `string | undefined`. I also added the node types for clarity though